plivo.py

```diff
@@ -13,6 +12,11 @@
 except ImportError:
     import simplejson as json
 
+try:
```

Comment:
This is just to make sure importing this file under python3 and using methods which don't use plivoxml will continue to work under python3.